hack/verify-jsonformat.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${OS_ROOT}/hack/common.sh"
+
+cd "${OS_ROOT}"
+
+json_files=$(find {api,examples,docs,images,plugins,test} -name "*.json")
+tmp_dir=$(mktemp -d --suffix=jsonformat)
+found=0
+
+set +e
+
+enable_formatting="${1:-""}"
+fix_formatting="${2:-""}"
+
+for f in $json_files; do
+  tmp_file="${tmp_dir}$(basename $f)"
+  go run ./hack/jsonformat.go ${f} > ${tmp_file}
+  result=$?
+  if [ "${enable_formatting}" == "--with-formatting" ]; then
```

Comment:
@bparees this will enable check that the JSON file is properly formatted (iow. the original JSON and the one jsonformat.go produce matches). There is a bug in Go (most likely) that causes big numbers to be parsed incorrectly which breaks this, so I made it optional (you can still check formatting but need to enable it). By default, this command just verifies JSON "errors" (not formatting).